include/game.h

```diff
@@ -250,6 +250,8 @@ struct Camera {
 
 extern struct Camera gCamera;
 
+#define PlayerIsAlive (!(gPlayer.moveState & MOVESTATE_DEAD))
```

Comment:
Cool!

Reply:
Yeah, it's occuring so often, I just thought this is much more readable!